pandas/tests/io/json/test_pandas.py

```diff
@@ -799,107 +798,92 @@ def test_series_non_unique_index(self):
-                    )
-                else:
-                    assert_series_equal(
-                        series,
-                        unser,
-                        check_names=False,
-                        check_index_type=check_index_type,
-                    )
-
-        def _check_all_orients(series, dtype=None, check_index_type=True):
-            _check_orient(
-                series, "columns", dtype=dtype, check_index_type=check_index_type
-            )
-            _check_orient(
-                series, "records", dtype=dtype, check_index_type=check_index_type
-            )
-            _check_orient(
-                series, "split", dtype=dtype, check_index_type=check_index_type
-            )
-            _check_orient(
-                series, "index", dtype=dtype, check_index_type=check_index_type
-            )
-            _check_orient(series, "values", dtype=dtype)
-
-            _check_orient(
-                series,
-                "columns",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "records",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "split",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "index",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-            _check_orient(
-                series,
-                "values",
-                dtype=dtype,
-                numpy=True,
-                check_index_type=check_index_type,
-            )
-
-        # basic
-        _check_all_orients(self.series)
+    def test_series_default_orient(self):
         assert self.series.to_json() == self.series.to_json(orient="index")
 
-        objSeries = Series(
+    @pytest.mark.parametrize("numpy", [True, False])
+    def test_series_roundtrip_simple(self, orient, numpy):
+        data = self.series.to_json(orient=orient)
+        result = pd.read_json(data, typ="series", orient=orient, numpy=numpy)
+        expected = self.series.copy()
+
+        if not numpy and PY35 and orient in ("index", "columns"):
+            expected = expected.sort_index()
+        if orient in ("values", "records"):
+            expected = expected.reset_index(drop=True)
+        if orient != "split":
+            expected.name = None
+
+        tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("dtype", [False, None])
+    @pytest.mark.parametrize("numpy", [True, False])
+    def test_series_roundtrip_object(self, orient, numpy, dtype):
```

Comment:
https://github.com/pandas-dev/pandas/pull/27838/files?file-filters%5B%5D=#r324901569 notwithstanding, this func corresponds to L885-L890 and LGTM